src/ProjectOrigin.Registry.IntegrationTests/PerformanceTests.cs

```diff
@@ -137,7 +151,7 @@ public async Task TestThroughput()
         Console.WriteLine($"Completed {completed.Count} transactions in {elapsedSeconds} seconds ({requestsPerSecond} requests per second).");
         Console.WriteLine($"-- Finished throughput test --");
 
-        requestsPerSecond.Should().BeGreaterThan(30); // based on througput test on github ~35
+        requestsPerSecond.Should().BeGreaterThan(150); // based on througput test on github ~170
```

Comment:
What exactly is tested here? could a situation ever occur where sent requests =/= requestsPerSecond?